### ISSUE TITLE
Does this repo have a license?

### ISSUE THREAD

**Issue by jcbhmr** [turtlemay/jsx-dom#15]
Hello,

I would like to fork this repo to add support for non-class-based custom elements like this:
```ts
function JustAFunction({ name }) {
    return html`<p>Hello ${name}!</p>`
}

class CustomClass extends HTMLElement {
    // ...
}
customElements.define("...", CustomClass)

html`
    <!-- Currently does not work, I want to fork and add this -->
    <${JustAFunction} name="world" />

    <!-- Works great! -->
    <${CustomClass} name="world" />
`
```

However, I do not see a license? I was under the impression (from my limited OSS experience; namely _none_) that I am supposed to a) include a `Forked from ${repo} under ${license}` and b) keep the original `license.txt` file?

I found that the `package.json` has a `license: "MIT"` directive, but I don't think that counts? I think that's just a hint to NPM? IDK.

Basically, I am asking _will you pls pls pls not sue me_ if I fork this, and/or can you add a license file of your choosing to the repo? Maybe this is unneeded, and I can just fork it and go on my way, but IDK man, I just wanted to check.

**Comment by jcbhmr** [comment]
While looking around for alternatives, I found some other repos that do something similar (JSX-to-DOM conversions) that I might fork/use instead:

- https://github.com/Lusito/tsx-dom -- [does not support custom elements](https://github.com/Lusito/tsx-dom/blob/master/src/index.ts#L58)
- https://github.com/proteriax/jsx-dom -- [does not support custom elements, only `React.Component` class](https://github.com/proteriax/jsx-dom/blob/master/src/jsx-dom.ts#L171)
- https://github.com/NewBuilding/dom-jsx -- [does not support custom elements](https://github.com/NewBuilding/dom-jsx/blob/master/src/h.ts#L11)

**Comment by jcbhmr** [comment]
This is no longer "important" to me personally, since I have found an alternative in forking  Lusito/tsx-dom. Closing since I found a solution.
